tsconfig.json

```diff
@@ -12,11 +12,11 @@
         ],
         "types": [
             "node",
-            "jquery",
+			"jquery",
             "Phaser"
```

Comment:
?

Reply:
i just pulled and its showing lines 15 and 18 as tabs, the rest are spaces. I'm fine to merge its just a little thing

Reply:
updated, indentation is now consistent